R/interface.R

```diff
@@ -357,8 +357,8 @@ neighbors <- function(graph, v, mode = c("out", "in", "all", "total")) {
   if (length(v) == 0) {
     stop("No vertex was specified")
   }
-  on.exit(.Call(R_igraph_finalizer))
-  res <- .Call(R_igraph_neighbors, graph, v - 1, as.numeric(mode)) + 1L
+
+  res <- neighbors_impl(graph, vid = v - 1, mode = mode) + 1L
```

Comment:
@maelle When you move to auto-generation, you don't usually need to correct for zero-based vs one-based indexing. Check the auto-generated sources to see what you can replace here: it will often be quite a bit more than just the `.Call`.

In this case you can pass `v` instead of `v-1`.

Note that the autogen R sources already do this:

```r
  if (igraph_opt("return.vs.es")) {
    res <- create_vs(, res)
  }
```

Therefore this should be removed from _below_.

I expect the tests to pass after this.